neqo-crypto/src/agent.rs

```diff
@@ -893,8 +892,12 @@ impl Client {
             // Ignore the token.
             return ssl::SECSuccess;
         }
-        let resumption = arg.cast::<Vec<ResumptionToken>>().as_mut().unwrap();
-        let len = usize::try_from(len).unwrap();
+        let Some(resumption) = arg.cast::<Vec<ResumptionToken>>().as_mut() else {
+            return ssl::SECFailure;
+        };
+        let Ok(len) = usize::try_from(len) else {
+            return ssl::SECFailure;
```

Comment:
When can we starting using Try?

Reply:
You mean https://doc.rust-lang.org/std/ops/trait.Try.html?

Reply:
Alternative:

```
let len = usize_try_from(len).map_err(|_| ssl::SECFailure)?;
```

Not sure what clippy prefers.